packages/mgt-chat/src/components/Chat/Chat.tsx

```diff
@@ -96,8 +109,24 @@ export const Chat = ({ chatId }: IMgtChatProps) => {
             </>
           ) : (
             <>
-              {chatState.status}
-              <Spinner />
+              {isLoading && (
+                <div className={styles.spinner}>
+                  <Spinner /> <br />
+                  {chatState.status}
+                </div>
+              )}
+              {chatState.status === 'initial' && (
```

Comment:
Adding @yejuntak. I feel we should have these statuses defined in the design and that they should probably not be status bars, but more like a "full screen" experience. For instance, Outlook uses something like this : 

![image](https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/7620955/c6975e4d-8403-46a6-adbb-d3130eb6530a)

@yejuntak Can you help designing these experiences and work with @musale for implementation?

Reply:
Absolutely I can help @musale , can you give me a chat/call on Teams and can you share me details?